tests/system/test_mem.c

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <fcntl.h>
+
+#include "devshell.h"
+
+void cmd_dev_null(int argc, char **argv) {
```

Comment:
Can you follow the 3-space indentation for this code, so that's aligned with the rest of the project?

Reply:
Thanks!